plaidml2/bridge/keras/__init__.py

```diff
@@ -1547,8 +1556,22 @@ def square(x):
 
 
 @_log_call
-def squeeze(x, axis):
-    return _KerasNode('squeeze', tensor=plaidml_op.squeeze(x.tensor, axis))
+def squeeze(x, axis=None):
+    if axis is None:
+        # define axis
+        axis = []
+        x_shape = shape(x)
```

Comment:
This one should be `int_shape` since a symbolic dimension won't `== 1` in python even if it evaluates to 1; using `int_shape` makes it clear that we're only squeezing along non-symbolic dimensions.